tests/torch/fx/test_models.py

```diff
@@ -90,7 +90,7 @@ def get_ref_metatypes_from_json(
         if not os.path.exists(json_parent_dir):
             os.makedirs(json_parent_dir)
         with safe_open(complete_path, "w") as file:
-            json.dump(model_metatypes, file)
+            json.dump(model_metatypes, file, indent=1)
```

Comment:
Looks better for reference files :+1: , but please use default value for indent - 4

Reply:
Done